.rtd-env-py3.yml

```diff
@@ -6,6 +6,8 @@ dependencies:
 - python=3.8
 - mpi4py
 - islpy
+- pip
```

Comment:
```suggestion
```

Reply:
I added this because of this warning message:
```
Warning: you have pip-installed dependencies in your environment file, but you do not list pip itself as one of your conda dependencies.  Conda may not use the correct pip to install your packages, and they may end up in the wrong place.  Please add an explicit pip dependency.  I'm adding one for you, but still nagging you.
```

Reply:
Huh, OK. TIL. Never mind then!